apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-detail-page.component.html

```diff
@@ -6,6 +6,10 @@ <h1 class="page-title">
     <a class="vo-link" [routerLink]="['/organizations/', vo.id]">
       {{vo.name}}
     </a>
+    <br>
+    <span class="vo-text" style="margin-left: 6.5em">
```

Comment:
Please, never write a style directly to an element like this. You should always prefer using some CSS classes instead. And I don't even like the position of the short name. It is nice, that it is aligned with the full name, but it will not work on all devices. It would be better if the short name would be positioned under the full name and on the left side. Also, the ID should be left where it was.